README.md

```diff
@@ -1,15 +1,20 @@
 ipymaterialui
 ===============================
 
-Jupyter Widgets based on React Material UI components
+Jupyter Widgets based on [Material-UI](https://material-ui.com/) which implement Google's 
+[Material Design Spec](https://material.io/) with [React](https://reactjs.org/).
 
 Installation
```

Comment:
I think it would be nice to have an overview of many widgets here (including a navigation drawer), possibly an animated gif.